### PR TITLE
Kk

### PR DESCRIPTION
**Pull request by mohana1809**
Hello

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by Armandd123** [comment]
j

**Comment by mohana1809** [comment]
Hi

**Comment by casebuiltfurniture** [comment]
Hope this one works

**Comment by casebuiltfurniture** [comment]
Ok

**Comment by casebuiltfurniture** [comment]
Ok

File: terms-of-service.txt [hunk 1]
```diff
@@ -1,7 +1,8 @@
 TERMS OF SERVICE LAST MODIFIED: 2018-04-17
 
 Tumblr is a U.S. company and subject to U.S. laws and jurisdiction. The original version of these Terms of Service (and any other terms, policies, or guidelines that we provide to you) are written in English. To the extent any translated version of these Terms of Service (or any other terms, policies, or guidelines that we provide to you) conflicts with the English version, the English version controls.
```
Comment (0806549105) [inline terms-of-service.txt]: Duplicate of #

**Comment by capriceclasssics** [comment]
Don't understand.

**Review by martinliesk — APPROVED** [review]

**Review by martinliesk — APPROVED** [review]

**Review by martinliesk — APPROVED** [review]

**Pull request closed without merging**